src/__tests__/interactions-helper.js

```diff
@@ -1,6 +1,7 @@
 import { within, userEvent, fireEvent } from "@storybook/testing-library";
 import { waitFor } from "@testing-library/react";
-import { getTestId, ELEMENT_TYPES as types, NAVIGATIONS_COMMANDS } from "../utils/test-utils";
+import { getTestId, ELEMENT_TYPES as types } from "../utils/test-utils";
+import { NavigationCommand as NAVIGATIONS_COMMANDS } from "../interactions-tests/constants";
```

Comment:
for backward compatibility (trying not to create a huge pr (will removed seperatly)